<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propVolTK</class>
 <widget class="QWidget" name="propVolTK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="volTK">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>0</y>
     <width>194</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>TK Volume</string>
   </property>
   <widget class="QTextEdit" name="info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinWinMin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>window bottom: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="addColorWidgetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add color filter</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>192</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>190</width>
       <height>159</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="spinWinMax">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>window top: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xBspin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="xEspin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yEspin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yBspin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zEspin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zBspin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>info</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinWinMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>winMinValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinWinMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>winMaxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addColorWidgetButton</sender>
   <signal>clicked()</signal>
   <receiver>propVolTK</receiver>
   <slot>addColorWidgetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xBspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>xBchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xEspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>xEchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yBspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>yBchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yEspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>yEchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zBspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>zBchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zEspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propVolTK</receiver>
   <slot>zEchanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>winMinValueChanged(int)</slot>
  <slot>winMaxValueChanged(int)</slot>
  <slot>addColorWidgetButtonClicked()</slot>
  <slot>xBchanged(int)</slot>
  <slot>yBchanged(int)</slot>
  <slot>zBchanged(int)</slot>
  <slot>xEchanged(int)</slot>
  <slot>yEchanged(int)</slot>
  <slot>zEchanged(int)</slot>
 </slots>
</ui>
